src/eleveldb_sup.erl

```diff
@@ -0,0 +1,36 @@
+%% -------------------------------------------------------------------
+%%
+%%  eleveldb_sup: Supervisor for eleveldb_metadata
+%%
+%% Copyright (c) 2017 Basho Technologies, Inc. All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(eleveldb_sup).
+
+-behaviour(supervisor).
+
+-export([start_link/0]).
+-export([init/1]).
+
+start_link() ->
+    supervisor:start_link({local, ?MODULE}, eleveldb_sup, []).
+
+init(_Args) ->
+    SupFlags = {one_for_one, 2, 5},
+    ChildSpecs = [{metadata, {eleveldb_metadata, start_link, []},
+                   permanent, brutal_kill, worker, [eleveldb_metadata]}],
```

Comment:
Why `brutal_kill`?  The process should be written to handle exit messages cleanly.